### PR TITLE
Fix styling for dynamic size changes

### PR DESCRIPTION
**Pull request by mtjandra**
Size changes should now react correctly for ie, edge, firefox and chrome. In both polymer 2 and 1.

### CODE REVIEW

File: src/d2l-profile-image-shared-styles.html [hunk 1]
```diff
@@ -5,36 +5,66 @@
...
 				display: inline-block;
-				--size: 84px;
+				width: 84px;
+				height: 84px;
+				--d2l-icon-height: 100%;
+				--d2l-icon-width: 100%;
 				border-radius: 8px;
 			}
+
+			@media screen and (-ms-high-contrast: none), (-ms-high-contrast: active) {
```
Comment (svanherk) [inline src/d2l-profile-image-shared-styles.html]: Are these specific styles for IE11?  If I remove this whole block, it seems to work fine, but I may be missing something.

Comment (mtjandra) [reply src/d2l-profile-image-shared-styles.html]: Did you try changing the sizes after it loaded in IE? In my experience IE would not respond to the size changes if I remember correctly

Comment (svanherk) [reply src/d2l-profile-image-shared-styles.html]: I did, and it looks to work in Polymer 1 at least

Comment (mtjandra) [reply src/d2l-profile-image-shared-styles.html]: Yup you're right I double checked. I probably didn't try deleting it after adding the css for `.shady-person` for IE I'll delete

**Pull request merged**